.gitlab-ci.yml

```diff
@@ -763,6 +763,26 @@ workflow:
 
 # New E2E related rules
 
+.on_e2e_or_fakeintake_changes_or_manual:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should go to https://github.com/DataDog/datadog-agent/blob/pducolin/ADXT-124-update-qa-labels-doc/.gitlab/dev_container_deploy/fakeintake.yml#L2-L15

Reply:
Nevermind